_includes/manuals/1.9/7.3_getting_help.md

```diff
@@ -8,3 +8,12 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
 
 * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
 * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
+
+### Gathering information
+* In order to troubleshoot and get relevant data use Foreman-debug specified in [genral support] (http://theforeman.org/support.html)
+* To upload the tarball to our public server via rsync use:
+
+        Foreman-debug -u
+
+  This is a write-only directory (readable only by Foreman core developers), please note that the rsync transmission is UNENCRYPTED.
```

Comment:
Stray space at the start of the line, and I'd split this into two sentences.

Consider adding the other info about filtering that are present in support.md.